tests/jax/huggingface/targets.jsonnet

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License,
+
+local text_classification = import 'glue.libsonnet';
+
+// Add new models here
+std.flattenArrays([
+  text_classification.configs,
+])
```

Comment:
Could we move out of the huggingface folder and into `tests/jax/latest` for now?

I like the idea though, might ask to move it out in a future PR but I would need to think about this for a bit